src/components/App.tsx

```diff
@@ -620,7 +620,9 @@ export class App extends React.Component<AppProps, AppState> {
 
     return <div className="fill">
       {this.state.toast &&
-          <Toast onDismiss={this.dismissToast.bind(this)} uri={this.state.toast}/>
+        <Toast onDismiss={this.dismissToast.bind(this)}
+               message={["Gist Created!", <a href={this.state.toast} target='_blank' className='gist-link'>Open in new tab.</a>]}
```

Comment:
Instead of passing an array for message parameter, you can pass in a JSX element. This could become simply:

```
message={<div>Gist Created! ... </div>}
```